pkg/controllermanager/controller/project/project/reconciler.go

```diff
@@ -309,7 +307,7 @@ func namespaceAnnotationsFromProject(project *gardencorev1beta1.Project) map[str
 	}
 }
 
-func (r *Reconciler) delete(ctx context.Context, log logr.Logger, project *gardencorev1beta1.Project) (reconcile.Result, error) {
+func (r *Reconciler) delete(ctx context.Context, log logr.Logger, project *gardencorev1beta1.Project, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
nit: In the other reconcilers, you have named this `req`, here it is `request`
```suggestion
func (r *Reconciler) delete(ctx context.Context, log logr.Logger, project *gardencorev1beta1.Project, req reconcile.Request) (reconcile.Result, error) {
```